framework/wazuh/core/wdb.py

```diff
@@ -54,6 +54,11 @@ def __query_input_validation(self, query):
                 (query_elements[2] == 'select',
                  'Wrong SQL query for Mitre database')
             ]
+        elif query_elements[sql_first_index] == 'rootcheck':
```

Comment:
The API? This is core framework, I dont understand this part completely.

Reply:
I used the same message as in line `#68`, but possibly both need to be updated.

https://github.com/wazuh/wazuh/blob/ce7a5686320a174db78a9e47850417bb48206d2f/framework/wazuh/core/wdb.py#L68